secret-providers/README.md

```diff
@@ -0,0 +1,33 @@
+# Camunda Connectors Secret Provider
+
+For AWS SaaS Clusters the AWS Secret Manager is used. For GCP SaaS Clusters the GCP Secret Manager is used.
+
+## Configuration
+
+Secret Provider that delegates to the GCP Security Manager.
+
+Requires the following properties being set,e.g. via application.properties:
+
+* `camunda.client.cloud.clusterId`: Cluster ID of Camunda SaaS cluster
+* `camunda.saas.secrets.projectId`: GCP project ID to use
+
+Optionally the following property can be set:
+* `camunda.saas.secrets.prefix`: If not set defaults to `connector-secrets`
```

Comment:
I think the contents were focused on using the secret providers independently of our SaaS setup. We should there not mention:
> For AWS SaaS Clusters the AWS Secret Manager is used. For GCP SaaS Clusters the GCP Secret Manager is used.
but treat the secret providers as generic implementations that can also be used outside of our SaaS offering in self-managed setups.